src/html/include/base.html

```diff
@@ -129,7 +129,7 @@
                 <p>Copyright &copy; SpongePowered 2014-2017</p>
                 <p id="fastly">Accelerated by<br>
                     <a href="https://www.fastly.com">
-                        <img src="assets/img/fastly.png" alt="Fastly" />
+                        <img src="assets/img/fastly.png" alt="Fastly" height="45" width="100">
```

Comment:
If height and width are set, the space required for the image is reserved when the page is loaded. Thus, the page load faster.